### PR TITLE
CIP-0036 | Adjust preamble and structure w.r.t CIP-0001

### LINKED ISSUE #604: CIP-0036 | Adjust preamble and structure w.r.t CIP-0001
Fixed by #732.

### PR DESCRIPTION
**Pull request by Ryun1**
Fixes #604.

#### Changes
- Removed `Voltaire` from the title as it is now clear that this will not be used for Voltaire.
- Moved rationale to `Rationale` section.
- Moved `Changelog` subsection to `Rationale` section.
- Added `Path to Active` section.
- Added links/missing words/missing grammar.
- Removed extra spaces and unused reference.

------

[Rendered Updated Proposal](https://github.com/Ryun1/CIPs/blob/remediate-36/CIP-0036/README.md)

### CODE REVIEW

**Comment by Ryun1** [comment]
Tagging current Catalyst team for approval @stevenj

File: CIP-0036/README.md [hunk 1]
```diff
@@ -266,8 +273,23 @@ Fund 10:
...
+- [ ] Have this registration format supported by three wallets/tools.
+  - [cardano-signer](https://github.com/gitmachtl/cardano-signer)
+
+### Implementation Plan
+
+- [x] Authors to provide test vector file.
+  - See the [schema file](./schema.cddl).
+- [x] Authors to provide CDDL schema file.
+  - See [test vector file](./test-vector.md).
+- [ ] Authors to test this format for security and compatibility with existing Catalyst infrastructure.
```
Comment (rphair) [inline CIP-0036/README.md:290]:
```suggestion
- [x] Authors to test this format for security and compatibility with existing Catalyst infrastructure.
```
I can confirm compatibility myself (I registered for & successfully voted in Fund 10 with CIP-36 compatible command line tooling) so maybe if @stevenj can confirm no known security holes then this box can be ticked as well.

**Review by rphair — APPROVED** [review]
Looks great, confirming title change is appropriate (with `/` less likely to break scripts).  Would welcome any author discussion (@stevenj) in this PR about how the implementations are going. 🤓

**Comment by rphair** [comment]
@Ryun1 @stevenj @gufmar - following up what what I mentioned I did in https://github.com/cardano-foundation/CIPs/pull/732#discussion_r1439651533 - does `cardano-signer` count as an `Implementor` if it generates JSON format metadata compliant with this CIP?
https://github.com/gitmachtl/cardano-signer#cip36-voting-keys-withoutwith-mnemonics

**Comment by Ryun1** [comment]
> does cardano-signer count as an Implementor if it generates JSON format metadata compliant with this CIP?

Yes! good spot, we should add to `Implementors` in the header

File: CIP-0036/README.md [hunk 2]
```diff
@@ -1,20 +1,28 @@
...
-Comments-URI: https://forum.cardano.org/t/cip-catalyst-registration-metadata-format/44038
+Title: Catalyst Registration Transaction Metadata Format (Updated)
+Category: Tools
 Status: Proposed
-Type: Standards
+Authors:
+  - Giacomo Pasini <giacomo.pasini@iohk.io>
+  - Kevin Hammond <kevin.hammond@iohk.io>
+  - Mark Stopka <mark.stopka@perlur.cloud>
+Implementors: []
```
Comment (rphair) [inline CIP-0036/README.md:10]:
```suggestion
Implementors:
  - cardano-signer <https://github.com/gitmachtl/cardano-signer>
```

**Review by rphair — COMMENTED** [review]
as per https://github.com/cardano-foundation/CIPs/pull/732#issuecomment-1874646464 (it was already marked in _Path to Active_)

**Review by KtorZ — APPROVED** [review]

**Comment by Ryun1** [comment]
Probably wasn't quite ready to merge this one, waiting on some context from the Catalyst team 
@stevenj please open a new PR to add the context we talked about

**Pull request merged**